core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -903,6 +900,7 @@ object ConsumerGroupCommand extends Logging {
       "Has the following scenarios to choose: --to-datetime, --by-period, --to-earliest, --to-latest, --shift-by, --from-file, --to-current. One scenario must be choose" + nl +
       "To define the scope use: --all-topics or --topic. . One scope must be choose, unless you use '--from-file' scenario"
     val DryRunDoc = "Only show results without executing changes on Consumer Groups. Supported operations: reset-offsets."
+    val ExecuteDoc = "Execute operation. Supported operations: reset-offsets."
```

Comment:
Should we update line 899 above mentioning `--dry-run` to plan, and then if nothing is specified it is the default option.